ui/app/components/job-status/allocation-status-row.hbs

```diff
@@ -55,7 +55,11 @@
     </div>
   {{/if}}
   {{#if @compact}}
-    {{@runningAllocs}}/{{@groupCountSum}}
+    {{#if (eq @allocationTallyMode "complete")}}
+      {{@completeAllocs}}/{{@groupCountSum}}
+    {{else}}
+      {{@runningAllocs}}/{{@groupCountSum}}
+    {{/if}}
```

Comment:
Felt it made more sense, in the context of a list of parameterized/periodic child jobs, which are going to be batch/sysbatch, to tally up the completed allocations rather than the running ones.

This is inconsistent with what we show on the jobs index page (running/expected), but contextually probably more relevant. I'm only feeling about 75% confident of this change so open to undoing it.